uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/SamlLoginIT.java

```diff
@@ -328,9 +328,42 @@ void contentTypes() {
     void simpleSamlPhpPasscodeRedirect() {
         createIdentityProvider(SAML_ORIGIN);
 
-        PasscodePage.assertThatRequestPasscode_goesToLoginPage(webDriver, baseUrl)
-                .assertThatSamlLink_goesToSamlLoginPage(SAML_ORIGIN)
-                .assertThatLogin_goesToPasscodePage(testAccounts.getUserName(), testAccounts.getPassword());
+        // Request passcode (redirects to login), perform SAML login, should redirect to passcode
+        LoginPage loginPage = PasscodePage.assertThatRequestPasscode_goesToLoginPage(webDriver, baseUrl);
+        
+        // Navigate to SAML endpoint
+        webDriver.get(String.format("%s/saml2/authenticate/%s", baseUrl, SAML_ORIGIN));
```

Comment:
same comment, why we are directly opening samlAuthUrl, saml login link you can control using web_domains